index.html

```diff
@@ -401,29 +401,11 @@
                 <h2>Firing events using the <code>PointerEvent</code> interface</h2>
                 <p>To <dfn>fire a pointer event name e</dfn> means to <dfn>fire an event named e</dfn> as defined in [[!DOM4]] with an event using the <a>PointerEvent</a> interface whose attributes are set as defined in <a href="#pointerevent-interface"><code>PointerEvent</code> Interface</a>.</p>
 
-                <p>Initialize the <code>composed</code> [[!WHATWG-DOM]] attribute to <code>true</code> for any pointer event.</p>
-
-                <p>Initialize the <code>bubbles</code> attribute for the event to <code>true</code> if the event name is</p>
-                <ul>
-                    <li><code>pointerdown</code></li>
-                    <li><code>pointerup</code></li>
-                    <li><code>pointercancel</code></li>
-                    <li><code>pointermove</code></li>
-                    <li><code>pointerover</code></li>
-                    <li><code>pointerout</code></li>
-                    <li><code>gotpointercapture</code></li>
-                    <li><code>lostpointercapture</code></li>
-                </ul>
-                <p>Initialize the <code>cancelable</code> attribute for the event to <code>true</code> if the event name is<p>
-                <ul>
-                    <li><code>pointerdown</code></li>
-                    <li><code>pointerup</code></li>
-                    <li><code>pointermove</code></li>
-                    <li><code>pointerover</code></li>
-                    <li><code>pointerout</code></li>
-                </ul>
-                
-                <p>Run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps for this <code>PointerEvent</code> if it is not <code>gotpointercapture</code> or <code>lostpointercapture</code>.
+                <p>For any pointer event initialize the <code>composed</code> ([[!WHATWG-DOM]]) attribute to <code>true</code> and <a href="https://www.w3.org/TR/uievents/#widl-UIEvent-detail"><code>detail</code></a> [[!DOM-LEVEL-3-EVENTS]] attribute to 0.</p>
+                <p>Initialize the <code>bubbles</code> and <code>cancelable</code> attributes for the event according to the <a href="#pointer-event-type-table">Pointer Events table</a>.</p>
+                <p>For <code>gotpointercapture</code> and <code>lostpointercapture</code> all the attributes except the ones defined in the table above should be the same as the Pointer Event that caused the user agent to run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> and fire the <code>gotpointercapture</code> and <code>lostpointercapture</code> events.</p>
```

Comment:
Replace "table above" with the link too. On my first reading, I scrolled up to find the buttons table!